README.md

```diff
@@ -58,6 +56,12 @@ If you're creating a new topic, keep the following in mind as well:
 | reference | Any reference-type article that isn't auto-generated. |
 | article | Use for community-contributed content (that is, anything from outside the engineering team or the content team at Microsoft. |
 
+## Merging to the live branch
+
+Periodically, about once a month, a manual RI is performance from `main` -> `live` branch.
```

Comment:
what's an RI?

Reply:
RI = Reverse Integration, https://learn.microsoft.com/en-us/azure/devops/repos/tfvc/branch-strategically?view=azure-devops#how-does-your-team-manage-code-while-it-introduces-multiple-changes-simultaneously-through-several-project-releases has a reasonable description.